cmake/FindYubiKey.cmake

```diff
@@ -17,8 +17,8 @@ find_path(YUBIKEY_CORE_INCLUDE_DIR yubikey.h)
 find_path(YUBIKEY_PERS_INCLUDE_DIR ykcore.h PATH_SUFFIXES ykpers-1)
 set(YUBIKEY_INCLUDE_DIRS ${YUBIKEY_CORE_INCLUDE_DIR} ${YUBIKEY_PERS_INCLUDE_DIR})
 
-find_library(YUBIKEY_CORE_LIBRARY yubikey)
-find_library(YUBIKEY_PERS_LIBRARY ykpers-1)
+find_library(YUBIKEY_CORE_LIBRARY NAMES yubikey.dll yubikey.so yubikey)
+find_library(YUBIKEY_PERS_LIBRARY NAMES ykpers-1.dll ykpers-1.so ykpers-1)
```

Comment:
Actually one more thing: when you add `.so`, you also need to add `lib` at the beginning.

Reply:
cmake prepends that automagically.

Reply:
No, only when you leave out the .so. I tested it, and yubikey.so can't be found, only libyubikey.so.